src/org/opendatakit/briefcase/reused/transfer/AggregateServer.java

```diff
@@ -54,10 +55,12 @@
 public class AggregateServer implements RemoteServer {
   private final URL baseUrl;
   private final Optional<Credentials> credentials;
+  private List<InputStream> fileStreams;
 
   public AggregateServer(URL baseUrl, Optional<Credentials> credentials) {
     this.baseUrl = baseUrl;
     this.credentials = credentials;
+    this.fileStreams = new ArrayList<>();
```

Comment:
```suggestion
  private final List<InputStream> fileStreams = new ArrayList<>();

  public AggregateServer(URL baseUrl, Optional<Credentials> credentials, Array) {
    this.baseUrl = baseUrl;
    this.credentials = credentials;
```

Reply:
I suggest making the list `final`, since we're not really want to support reassigning a new value. Also, since this is a property belonging to the internal workings of the class, I think it's OK to assign its value directly in the property declaration.